docs/studios/unity/unity-sdk/RGFindUtils.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: 'RGFindUtils'
+---
+
+# RGFindUtils
+
+`RGFindUtils` is a singleton providing a set of utility functions for finding GameObjects. This
+is extremely useful when implementing [`RGAction`](./RGAction)s that need to find other GameObjects in the scene.
+
+The instance of `RGFindUtils` is automatically created when the scene is loaded, and can be accessed
+with the following function:
+
+```cs
+RGFindUtils.Instance
```

Comment:
Not really a not for these docs, but in general - some of our singletons follow this pattern while others are accessed using a `GetInstance()` method. We should consider making these consistent

Reply:
Yeah agreed... I'll make sure to write that down somewhere! @rcornwall-reg what do you find is most common? Or I think maybe you even mentioned not going with a singleton and going with a different approach?